.env.template

```diff
@@ -0,0 +1,21 @@
+# .env file
+################################################################################
+### GPT-LLAMA - GENERAL SETTINGS
+################################################################################
+
+## [REQUIRED] LLAMA_PATH - The absolute path of llama.cpp installation directory.
+LLAMA_PATH="/mnt/set/this/to/llama.cpp"
```

Comment:
i actually avoided this so we can put this in the API KEY. less work for the end user to set up. i would prefer if we didn't move the llama_path to .env for now